pkg/suggestion/v1beta1/internal/search_space.py

```diff
@@ -36,15 +37,35 @@ def convert(experiment):
             search_space.goal = constant.MIN_GOAL
         for p in experiment.spec.parameter_specs.parameters:
             search_space.params.append(
-                HyperParameterSearchSpace.convertParameter(p))
+                HyperParameterSearchSpace.convert_parameter(p))
         return search_space
 
+    @staticmethod
+    def convert_to_combinations(search_space):
+        combinations = {}
+
+        for parameter in search_space.params:
+            if parameter.type == INTEGER:
+                combinations[parameter.name] = range(int(parameter.min), int(parameter.max)+1, int(parameter.step))
+            elif parameter.type == DOUBLE:
+                if parameter.step == "" or parameter.step is None:
+                    raise Exception("Param {} step is nil".format(parameter.name))
```

Comment:
Should we also say that: "For discrete search space all parameters must include step" ?

Reply:
Sounds good. It helps to debug `parameters`.